models/stg_snapchat.yml

```diff
@@ -356,4 +362,136 @@ models:
+          combination_of_columns:
+            - source_relation
+            - campaign_id
+            - date_day
+            - country 
+    columns:
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
+      - name: campaign_id
+        description: ID of campaign
+        tests:
+          - not_null
+      - name: country
+        description: The country that the report relates to.
+        tests:
+          - not_null
+      - name: date_day
+        description: The day that the report relates to.
+        tests:
+          - not_null
+      - name: attachment_quartile_1
+        description: Number of times your Long Form Video was viewed to 25% 
+      - name: attachment_quartile_2
+        description: Number of times your Long Form Video was viewed to 50% 
+      - name: attachment_quartile_3
+        description: Number of times your Long Form Video was viewed to 75% 
+      - name: attachment_total_view_time
+        description: Total time Snapchatters spent on the attachment (HH:MM:SS)
+      - name: attachment_view_completion
+        description: The number of times your Long Form Video was viewed to completion (97%)
+      - name: quartile_1
+        description: The number of times your video ad was viewed to 25% 
+      - name: quartile_2
+        description: The number of times your video ad was viewed to 50% 
+      - name: quartile_3
+        description: The number of times your video ad was viewed to 75% 
+      - name: saves
+        description: Number of times components of the campaign were saved to Memories
+      - name: shares
+        description: Number of times components of the campaign were shared in a Chat or Story
+      - name: screen_time
+        description: The number of seconds spent watching your ad across all paid impressions. Screen time starts recording as soon as the media is fully rendered on the device and the autoplay video starts
+      - name: video_views
+        description: The number of times your video was watched at least 2 seconds or swiped up, whichever comes first
+      - name: view_completion
+        description: The number of times your video ad was viewed to completion (97% )
+      - name: view_time
+        description: The number of seconds spent watching your ad across all users reached.
+      - name: impressions
+        description: The number of impressions for an ad.
+      - name: spend
+        description: The amount of spend for an ad.
+      - name: swipes
+        description: The number of swipes for an ad. Swipes are when your ad was swiped up on or the CTA was tapped to view the attachment below.
+      - name: conversion_purchases_value
+        description: The total value of purchases attributed to an ad.
+      - name: conversion_purchases
+        description: The total number of purchases attributed to an ad for the day of the record.
+      - name: _fivetran_synced
+        description: "{{ doc('_fivetran_synced') }}"
```

Comment:
We actually don't include this in the staging model, so we can remove

Reply:
ah yes thanks for noticing that